src/main/python/janusgraph_python/core/datatypes/GeoShape.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from .Point import Point
+from .Circle import Circle
+
+
+class GeoShape(object):
+    """
+    This class is the super class for all GeoShapes.
+    """
+
+    @staticmethod
+    def Point(longitude, latitude):
+        """
+        This is wrapper method to call the actual Geographical Point class.
```

Comment:
This comment only explains the implementation instead of telling the user what the method does. How about something like _Creates a Point with the given coordinates._ instead?

Reply:
I've updated docs for Point based on my previous comments for Circle class. The docs and methods are analogous.